CHANGELOG-1.1.md

```diff
@@ -0,0 +1,29 @@
+# Changelog since v1.0.1
+
+## Deprecations
+
+* Command line flag `-connection-timeout` is deprecated and has no effect.
+
+## Actions needed
+
+* Cluster admins must update RBAC rules. Previously, the cluster driver registrar created CSIDriver in `csi.storage.k8s.io/v1alpha1` API group, now it creates it in `csi.k8s.io/v1beta1`. See [deploy/kubernetes/rbac.yaml](deploy/kubernetes/rbac.yaml) for example.
+
+## Notable Features
+
+* The cluster driver registrar now tries to connect to CSI driver indefinitely. ([#32](https://github.com/kubernetes-csi/cluster-driver-registrar/pull/32))
+
+* `CSIDriver` object is now beta and has been moved from `csi.storage.k8s.io/v1alpha1` to `csi.k8s.io/v1beta1`. New RBAC rules are required!
+
+## Other notable changes
+
+* Use distroless as base image ([#39](https://github.com/kubernetes-csi/cluster-driver-registrar/pull/39))
```

Comment:
Nice job on all the comments. Did you do this by hand?

Reply:
I get PR numbers from all merge commits since the last release + get titles of these PRs. + some manual editing.

Stay tuned for https://github.com/kubernetes-csi/docs/issues/142!